Created a category : Array ADT.

saranshkhulbe7 opened the pull request:
In this I have created a new category : Array ADT.

Tasks done :

1. Updated Readme.md file.
2. Added introduction for Abstract Datatypes (ADT).
3. Added some programs of Array ADT.

saranshkhulbe7 committed:
Added append function for Array ADT.

saranshkhulbe7 committed:
Created into file for abstract datatype.

saranshkhulbe7 committed:
updated intro file

saranshkhulbe7 committed:
It is the first method of forming array ADT.

saranshkhulbe7 committed:
It is the 2nd method of declaring array ADT.

saranshkhulbe7 committed:
Update Readme.md

saranshkhulbe7 committed:
Array ADT Basic Functions file added.

This file contains the basic operations that are usually performed on Array ADT such as :
1. Max() : returns the largest element in the array.
2. Min() : returns the smallest element in the array.
3. Sum() : returns the sum of all the elements in the array.
4 Avg() : returns the average of all the elements in the array.

saranshkhulbe7 committed:
updated Array ADT Basic Functions

saranshkhulbe7 committed:
Created Binary Search Iterative version file.

This file illustrates Binary Search by Iterative Method.

saranshkhulbe7 committed:
Binary Search Recursive Version file.

This file illustrates Binary Search Using Recursion.

sahmad737 reviewed (commented):
Please try to add some comments and in ArrayDeclaration1 check for following. If the number of elements in array is greater than size of array still it works, and it neglects the size you have entered. 

Also please add comments to all file so its easier to understand.

saranshkhulbe7 committed:
Array_Declaration1.cpp updated

Exception Handled : if number of elements is > Array size
then "The number of elements cannot be more than the Array size" is printed.

saranshkhulbe7 committed:
Added Comments

saranshkhulbe7 committed:
Added Comments

saranshkhulbe7 committed:
Comments Added

saranshkhulbe7 committed:
Comments Added

saranshkhulbe7 committed:
Comments Added

saranshkhulbe7 committed:
Comments Added

saranshkhulbe7 commented:
In this I have created a new category : Array ADT.

Tasks done :

1. Updated Readme.md file.
2. Added introduction for Abstract Datatypes (ADT).
3. Added some programs of Array ADT.
4. Handled Exception of ArrayDeclaration1.cpp
5. Added comments to the code.
